services/hanke-service/src/main/resources/db/changelog/changesets/create-hankegeometriat-tables.sql

```diff
@@ -0,0 +1,68 @@
+--liquibase formatted sql
+--changeset teemu:create-hankegeometry-tables comment:Create Hanke Geometry tables
+
+CREATE EXTENSION IF NOT EXISTS postgis;
+
+CREATE TABLE IF NOT EXISTS HankeGeometriat
```

Comment:
IF NOT EXISTS  is dangerous. This could e.g. in some situations lead to old geometry tables in the database, while Liquibase thinks the changeset was run correctly (with newer structure).  To match how the Liquibase's own changes would work, there should be no IF NO EXISTS  (except maybe on that postgis extension thing, since that is simply on/off).
To be polite about it, one could add some kind of pre-check, but I don't know how to make that into a pure SQL changeset file.